Authentication Methods Registration Campaign Graph Docs (#12112)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/authenticationmethodspolicies-overview.md

* Define the types of FIDO2 security keys that can be used in the Azure AD tenant.
* Define the users or groups of users who are allowed to use FIDO2 Security Keys or Passwordless Phone Sign-in to sign in to Azure AD.

> * Define the users or groups of users who are to be nudged to set up the Microsoft Authenticator for MFA using push notifications.

Reviewer: ```suggestion
* Define the users or groups of users who should be reminded to set up the Microsoft Authenticator for MFA using push notifications.
```

|[passwordlessmicrosoftauthenticatorauthenticationmethodconfiguration](passwordlessmicrosoftauthenticatorauthenticationmethodconfiguration.md) (deprecated)|Define users who can use Passwordless Phone Sign-in to sign in to Azure AD.|
|[temporaryaccesspassauthenticationmethodconfiguration](temporaryaccesspassauthenticationmethodconfiguration.md)|Define users who can use Temporary Access Pass to sign in to Azure AD.|

> ## Policies available to push users to set up particular authentication methods:

Reviewer: ```suggestion
## Policies available to push users to set up authentication methods:
```

|Policy       | Description |
|:---------------------------|:------------|

> |[authenticationMethodsRegistrationCampaign](authenticationmethodsregistrationcampaign.md)| Define users who can be nudged to set up a particular authentication method (currently only supported for the Microsoft Authenticator).|

Reviewer: ```suggestion
|[authenticationMethodsRegistrationCampaign](authenticationmethodsregistrationcampaign.md)| Define users who should be reminded to set up a particular method (only supported for the Microsoft Authenticator).|
```
